packages/app/src/components-styled/content-header/index.tsx

```diff
@@ -83,13 +88,19 @@ const ReferenceBox = styled(Box)(
   css({
     maxWidth: '30em',
     marginRight: 3,
-    flex: asResponsiveArray({ md: '1 1 auto', lg: '1 1 60%' })
+    flex: asResponsiveArray({ md: '1 1 auto', lg: '1 1 60%' }),
   })
 );
 
 const MetadataBox = styled(Box)(
   css({
-    flex: asResponsiveArray({ md: '1 1 auto', lg: '1 1 40%' })
+    flex: asResponsiveArray({ md: '1 1 auto', lg: '1 1 40%' }),
+  })
+);
+
+const PointerEventsBox = styled(Box)(
```

Comment:
This can be a simple div
```suggestion
const PointerEventsBox = styled.div(
```